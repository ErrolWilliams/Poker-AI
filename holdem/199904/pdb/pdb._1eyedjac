_1eyedjac 924371884  9  5 Q   -     -     -         1000    0    0 
