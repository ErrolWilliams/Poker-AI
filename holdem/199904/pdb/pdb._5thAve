_5thAve   923015323  4  3 f   -     -     -         1339    0    0 
_5thAve   923015355  5  3 f   -     -     -         1339    0    0 
_5thAve   923015409  6  3 c   b     b     -         1339   40  100 
_5thAve   923015444  7  3 f   -     -     -         1399    0    0 
_5thAve   923015483  7  2 Bk  kc    k     b         1399   40  130 As 5h
_5thAve   923015555  8  1 Bc  kc    kc    kf        1489   50    0 
_5thAve   923015615  8  8 f   -     -     -         1439    0    0 
_5thAve   923015659  9  8 f   -     -     -         1439    0    0 
_5thAve   923015717  9  7 f   -     -     -         1439    0    0 
_5thAve   923015785 10  7 f   -     -     -         1439    0    0 
_5thAve   923015818 10  6 f   -     -     -         1439    0    0 
_5thAve   923015851 10  5 f   -     -     -         1439    0    0 
_5thAve   923015905 10  4 c   c     kc    br        1439  100  240 Qh Ks
_5thAve   923015954 10  3 f   Q     -     -         1579    0    0 
_5thAve   923096107 10  1 Bc  bc    kc    kf        1579   60    0 
_5thAve   923096164 10 10 f   -     -     -         1519    0    0 
_5thAve   923096210 10  9 f   -     -     -         1519    0    0 
_5thAve   923096255 10  8 c   kc    kc    kc        1519   60  205 8s 8d
_5thAve   923096316 10  7 f   -     -     -         1664    0    0 
_5thAve   923096348 10  6 c   c     c     c         1664   60    0 Ac 9c
_5thAve   923096383 10  5 c   f     -     -         1604   20    0 
_5thAve   923096426 10  4 f   -     -     -         1584    0    0 
_5thAve   923096475 10  3 f   -     -     -         1584    0    0 
_5thAve   923096521 10  2 Bf  -     -     -         1584   10    0 
_5thAve   923096597 10  1 Bf  -     -     -         1574    5    0 
_5thAve   923096664  9  9 c   c     k     k         1569   20    0 Qh Js
_5thAve   923096713  9  8 f   -     -     -         1549    0    0 
_5thAve   923096754  9  7 c   kc    kf    -         1549   20    0 
_5thAve   923096826  9  6 f   -     -     -         1529    0    0 
_5thAve   923096885  9  5 f   -     -     -         1529    0    0 
_5thAve   923096976 10  5 f   -     -     -         1529    0    0 
_5thAve   923097032 10  4 f   -     -     -         1529    0    0 
_5thAve   923097080 10  3 c   kf    -     -         1529   10    0 
_5thAve   923097118  9  2 Bf  -     -     -         1519   10    0 
_5thAve   923097183 10  1 Bf  -     -     -         1509    5    0 
_5thAve   923097268 10 10 f   -     -     -         1504    0    0 
_5thAve   923097324 10  9 f   -     -     -         1504    0    0 
_5thAve   923097363 10  9 f   -     -     -         1504    0    0 
_5thAve   923097425 10  7 c   c     f     -         1504   20    0 
_5thAve   923097504 10  6 f   -     -     -         1484    0    0 
_5thAve   923097609 10  5 cc  k     kc    kf        1484   40    0 
_5thAve   923097735  9  4 f   -     -     -         1444    0    0 
_5thAve   923097827  9  3 fQ  -     -     -         1444    0    0 
_5thAve   923438391 10  5 f   -     -     -         2579    0    0 
_5thAve   923438446 10  4 f   -     -     -         2579    0    0 
_5thAve   923438512 10  3 c   kc    kc    kc        2579   80    0 Kc Kd
_5thAve   923438572 10  2 Bc  kc    kc    br        2499  140  400 Tc Jc
_5thAve   923438659  9  1 Bf  -     -     -         2759    5    0 
_5thAve   923438736  9  9 c   f     -     -         2754   10    0 
_5thAve   923438824 10  9 f   -     -     -         2744    0    0 
_5thAve   923438895  9  7 cc  kc    kf    -         2744   40    0 
_5thAve   923438981 10  7 f   -     -     -         2704    0    0 
_5thAve   923439063 10  6 c   kf    -     -         2704   10    0 
_5thAve   923439155 10  5 f   -     -     -         2694    0    0 
_5thAve   923439201 10  4 f   -     -     -         2694    0    0 
_5thAve   923439261 10  3 f   -     -     -         2694    0    0 
_5thAve   923439324  9  2 Bf  -     -     -         2694   10    0 
_5thAve   923439362 10  1 Bf  -     -     -         2684    5    0 
_5thAve   923439412 10 10 f   -     -     -         2679    0    0 
_5thAve   923439502 10  9 f   -     -     -         2679    0    0 
_5thAve   923439616 10  8 f   -     -     -         2679    0    0 
_5thAve   923439661  9  6 f   -     -     -         2679    0    0 
_5thAve   923439715 10  6 f   -     -     -         2679    0    0 
_5thAve   923439765 10  5 f   -     -     -         2679    0    0 
_5thAve   923439830 10  4 f   -     -     -         2679    0    0 
_5thAve   923439894 10  3 f   -     -     -         2679    0    0 
_5thAve   923439985  9  2 Bk  kc    kc    kf        2679   40    0 
_5thAve   923440044  8  1 Bf  -     -     -         2639    5    0 
_5thAve   923440100  9  9 f   -     -     -         2634    0    0 
_5thAve   923440148  9  8 f   Q     -     -         2634    0    0 
_5thAve   923937840  9  4 f   -     -     -         2914    0    0 
_5thAve   923937909  9  3 f   -     -     -         2914    0    0 
_5thAve   923937929  8  2 Bf  -     -     -         2914   10    0 
_5thAve   923937944  8  1 Bf  -     -     -         2904    5    0 
_5thAve   923937978  8  8 c   c     k     bc        2899   60    0 Qs Jd
_5thAve   923938026  9  8 f   -     -     -         2839    0    0 
_5thAve   923938035  9  7 f   -     -     -         2839    0    0 
_5thAve   923938074  9  6 f   -     -     -         2839    0    0 
_5thAve   923938115  9  5 f   -     -     -         2839    0    0 
_5thAve   923938144  8  3 f   -     -     -         2839    0    0 
_5thAve   923938154  8  2 Bf  -     -     -         2839   10    0 
_5thAve   923938231  8  1 Bf  -     -     -         2829    5    0 
_5thAve   923938261  8  8 f   -     -     -         2824    0    0 
_5thAve   923938287  8  7 cc  c     f     -         2824   40    0 
_5thAve   923938349  8  6 f   -     -     -         2784    0    0 
_5thAve   923938410  9  6 f   -     -     -         2784    0    0 
_5thAve   923938441  9  4 f   -     -     -         2784    0    0 
_5thAve   923938458  9  3 f   -     -     -         2784    0    0 
_5thAve   923938502 10  3 f   -     -     -         2784    0    0 
_5thAve   923938560 10  2 B   -     -     -         2784   10   15 
_5thAve   923938571 10  1 Bf  -     -     -         2789    5    0 
_5thAve   923938620 10 10 f   -     -     -         2784    0    0 
_5thAve   923938661 10  9 c   k     c     r         2784   70  140 
_5thAve   923938717  9  7 f   -     -     -         2854    0    0 
_5thAve   923938810 10  7 f   -     -     -         2854    0    0 
_5thAve   923938860 10  6 c   k     k     kc        2854   30  105 3s 3d
_5thAve   923938963 10  3 Q   -     -     -         2929    0    0 
_5thAve   923945735  6  2 Bf  -     -     -         2929   10    0 
_5thAve   923945744  6  1 Bf  -     -     -         2919    5    0 
_5thAve   923945750  6  6 cc  c     c     c         2914   70  170 Ts Ks
_5thAve   923945788  6  5 f   -     -     -         3014    0    0 
_5thAve   923945828  5  3 fQ  -     -     -         3014    0    0 
_5thAve   924119130  9  5 cc  c     c     c         3279   70    0 Th Jh
_5thAve   924119169  9  4 f   -     -     -         3209    0    0 
_5thAve   924119196  9  4 cc  cc    k     kf        3209   40    0 
_5thAve   924119330  9  2 Bf  -     -     -         3169   10    0 
_5thAve   924119417 10  1 Bc  k     b     b         3159   50  150 9d Ad
_5thAve   924119513 10 10 f   -     -     -         3259    0    0 
_5thAve   924119575 10  9 f   -     -     -         3259    0    0 
_5thAve   924119690  9  8 f   -     -     -         3259    0    0 
_5thAve   924119726  8  7 f   -     -     -         3259    0    0 
_5thAve   924119808  8  6 f   -     -     -         3259    0    0 
_5thAve   924119832  8  5 f   -     -     -         3259    0    0 
_5thAve   924119874  9  4 f   -     -     -         3259    0    0 
_5thAve   924119917  8  3 cc  k     kc    k         3259   40  145 2c 2h
_5thAve   924119962  8  2 Bk  c     f     -         3364   20    0 
_5thAve   924119989  8  1 Bf  -     -     -         3344    5    0 
_5thAve   924120017  9  9 f   -     -     -         3339    0    0 
_5thAve   924120052  9  2 Q   -     -     -         3339    0    0 
_5thAve   924266201  8  8 cc  c     f     -         3709   40    0 
_5thAve   924266248  9  7 c   k     k     f         3669   10    0 
_5thAve   924266288 10  6 c   k     f     -         3659   10    0 
_5thAve   924266320 10  5 f   -     -     -         3649    0    0 
_5thAve   924266362 10  4 f   -     -     -         3649    0    0 
_5thAve   924266399 10  3 f   -     -     -         3649    0    0 
_5thAve   924266415 10  2 Bc  kc    kc    k         3649   50    0 Kc Qh
_5thAve   924266464  9  1 Bc  kc    b     b         3599   60  120 
_5thAve   924266497  9  9 f   -     -     -         3659    0    0 
_5thAve   924266559  9  8 f   -     -     -         3659    0    0 
_5thAve   924266588  9  7 f   -     -     -         3659    0    0 
_5thAve   924266643  9  6 f   -     -     -         3659    0    0 
_5thAve   924266687  9  5 f   -     -     -         3659    0    0 
_5thAve   924266720 10  5 cc  kc    kf    -         3659   50    0 
_5thAve   924266787 10  4 ccc kcc   kc    kf        3609  100    0 
_5thAve   924266853 10  3 f   -     -     -         3509    0    0 
_5thAve   924266880 10  2 Bk  kf    -     -         3509   10    0 
_5thAve   924266914  9  1 Bf  -     -     -         3499    5    0 
_5thAve   924266973  9  9 c   f     -     -         3494   10    0 
_5thAve   924267022  9  8 f   -     -     -         3484    0    0 
_5thAve   924267049  8  6 f   -     -     -         3484    0    0 
_5thAve   924267077  8  6 f   -     -     -         3484    0    0 
_5thAve   924267117  7  5 f   -     -     -         3484    0    0 
_5thAve   924267139  7  4 c   k     k     k         3484   10    0 Ac 6d
_5thAve   924267176  7  3 f   -     -     -         3474    0    0 
_5thAve   924267201  7  2 Bk  c     kf    -         3474   20    0 
_5thAve   924267244  7  1 Bf  -     -     -         3454    5    0 
_5thAve   924267272  7  7 f   -     -     -         3449    0    0 
_5thAve   924267299  7  6 f   -     -     -         3449    0    0 
_5thAve   924267305  7  5 f   -     -     -         3449    0    0 
_5thAve   924267337  8  5 f   -     -     -         3449    0    0 
_5thAve   924267371  8  4 c   k     f     -         3449   10    0 
_5thAve   924267438  8  3 f   -     -     -         3439    0    0 
_5thAve   924267472  8  2 Bf  -     -     -         3439   10    0 
_5thAve   924267477  8  1 Bc  k     k     kf        3429   10    0 
_5thAve   924267534  7  3 Q   -     -     -         3419    0    0 
_5thAve   924271639  9  7 fQ  -     -     -         3419    0    0 
_5thAve   924271812 10  3 f   -     -     -         3419    0    0 
_5thAve   924271884 10  2 Bk  bc    k     b         3419   50  150 7d Kh
_5thAve   924271975 10  1 Bc  kc    k     k         3519   20    0 5h 5d
_5thAve   924272036 10 10 f   -     -     -         3499    0    0 
_5thAve   924272062 10  9 f   -     -     -         3499    0    0 
_5thAve   924272153 10  5 Q   -     -     -         3499    0    0 
_5thAve   924306816  9  8 c   f     -     -         3452   20    0 
_5thAve   924306860  9  7 f   -     -     -         3432    0    0 
_5thAve   924306874  9  6 f   -     -     -         3432    0    0 
_5thAve   924306911 10  5 c   kc    kc    bc        3432   80  205 Jh Qh
_5thAve   924306969 10  4 c   c     f     -         3557   30    0 
_5thAve   924307036 10  3 cc  kc    k     kf        3527   30    0 
_5thAve   924307102 10  2 Bk  cc    kc    kc        3497   70  260 Ah 4d
_5thAve   924307165  9  1 Bc  kcc   kf    -         3687   40    0 
_5thAve   924307217  8  8 f   -     -     -         3647    0    0 
_5thAve   924307280  8  7 f   -     -     -         3647    0    0 
_5thAve   924307339  8  5 c   c     k     b         3647   40  130 
_5thAve   924307397  8  4 f   -     -     -         3737    0    0 
_5thAve   924307419  8  3 c   k     c     k         3737   30    0 9c Ac
_5thAve   924307459  8  2 Bk  b     b     -         3707   40   80 
_5thAve   924307490  7  1 BQ  -     -     -         3747    5    0 
_5thAve   924309317  9  8 cc  f     -     -         3187   40    0 
_5thAve   924309385  9  7 f   -     -     -         3147    0    0 
_5thAve   924309445 10  6 c   c     c     f         3147   50    0 
_5thAve   924309500 10  5 c   c     c     c         3097   70  195 7s Ks
_5thAve   924309535 10  4 fQ  -     -     -         3222    0    0 
_5thAve   924403090  9  5 f   -     -     -         3282    0    0 
_5thAve   924403140 10  5 cc  f     -     -         3282   20    0 
_5thAve   924403229 10  4 f   -     -     -         3262    0    0 
_5thAve   924403318 10  3 f   -     -     -         3262    0    0 
_5thAve   924403432 10  2 Bf  -     -     -         3262   10    0 
_5thAve   924403498 10  1 Bf  -     -     -         3252    5    0 
_5thAve   924403578 10 10 f   -     -     -         3247    0    0 
_5thAve   924403662 10  9 c   k     f     -         3247   10    0 
_5thAve   924403724 10  8 cc  kc    kc    kc        3237   70  195 Ah Ad
_5thAve   924403786 10  3 Q   -     -     -         3362    0    0 
_5thAve   925319297  9  6 f   -     -     -         3027    0    0 
_5thAve   925319328  9  5 f   -     -     -         3027    0    0 
_5thAve   925319369  9  4 f   -     -     -         3027    0    0 
_5thAve   925319422  9  3 f   -     -     -         3027    0    0 
_5thAve   925319461  9  2 Bc  kc    bc    b         3027   90  205 7h 7s
_5thAve   925319510 10  1 Br  kc    kc    kc        3142   70  180 Ad 8s
_5thAve   925319569 10 10 c   c     bc    c         3252   90    0 Ts Js
_5thAve   925319619 10  9 cc  c     c     k         3162   70    0 Tc Jh
_5thAve   925319708 10  8 f   -     -     -         3092    0    0 
_5thAve   925319754 10  7 c   c     c     k         3092   40  155 8c Ad
_5thAve   925319899 10  6 c   f     -     -         3207   10    0 
_5thAve   925319950 10  5 f   -     -     -         3197    0    0 
_5thAve   925320002 10  4 f   -     -     -         3197    0    0 
_5thAve   925320069 10  3 c   c     cc    kf        3197   60    0 
_5thAve   925320144 10  2 BQ  -     -     -         3137   10    0 
