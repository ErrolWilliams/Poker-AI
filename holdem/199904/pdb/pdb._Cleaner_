_Cleaner_ 925516793  3  3 cc  c     f     -          650   30    0 
_Cleaner_ 925516818  3  2 Br  b     brc   c          620  140    0 Kc 9d
_Cleaner_ 925516846  3  1 Bf  -     -     -          480    5    0 
_Cleaner_ 925516855  3  3 f   -     -     -          475    0    0 
_Cleaner_ 925516861  3  2 Bf  -     -     -          475   10    0 
_Cleaner_ 925516867  3  1 Bf  -     -     -          465    5    0 
_Cleaner_ 925516869  4  3 cc  b     r     -          460   70  140 
_Cleaner_ 925516898  5  2 Br  b     b     b          530   70  160 Qh Tc
_Cleaner_ 925516925  6  1 Br  b     kc    brc        620  130    0 Qc Tc
_Cleaner_ 925516962  6  6 f   -     -     -          490    0    0 
_Cleaner_ 925516983  6  5 f   -     -     -          490    0    0 
_Cleaner_ 925516986  6  4 cc  k     f     -          490   20    0 
_Cleaner_ 925517025  6  3 cc  b     b     -          470   50  105 
_Cleaner_ 925517053  7  2 Bc  f     -     -          525   20    0 
_Cleaner_ 925517070  7  1 Bc  kf    -     -          505   10    0 
_Cleaner_ 925517121  7  7 f   -     -     -          495    0    0 
_Cleaner_ 925517157  7  6 f   -     -     -          495    0    0 
_Cleaner_ 925517191  7  5 cc  k     k     k          495   20   80 8c 3c
_Cleaner_ 925517232  6  4 c   r     br    b          555  120    0 9c Jc
_Cleaner_ 925517279  6  4 cc  kf    -     -          435   20    0 
_Cleaner_ 925517317  5  2 Bc  b     -     -          415   30   55 
_Cleaner_ 925517334  4  1 Bf  -     -     -          440    5    0 
_Cleaner_ 925517372  4  4 f   -     -     -          435    0    0 
_Cleaner_ 925517391  4  3 f   -     -     -          435    0    0 
_Cleaner_ 925517405  4  2 Bf  -     -     -          435   10    0 
_Cleaner_ 925517422  4  1 Bf  -     -     -          425    5    0 
_Cleaner_ 925517441  4  4 c   k     k     b          420   30   50 
_Cleaner_ 925517483  4  3 c   r     -     -          440   30   55 
_Cleaner_ 925517502  4  2 Bc  k     k     b          465   40    0 As 2d
_Cleaner_ 925517551  4  1 Bf  -     -     -          425    5    0 
_Cleaner_ 925517576  4  4 r   c     c     k          420   80    0 Ad Qh
_Cleaner_ 925517618  5  3 f   -     -     -          340    0    0 
_Cleaner_ 925517678  6  2 Bc  f     -     -          340   20    0 
_Cleaner_ 925517732  6  1 Br  brc   k     b          320   80  190 6s 6h
_Cleaner_ 925517783  6  6 f   -     -     -          430    0    0 
_Cleaner_ 925517822  6  5 c   c     k     f          430   20    0 
_Cleaner_ 925517858  6  4 c   kc    kf    -          410   20    0 
_Cleaner_ 925517902  6  3 f   -     -     -          390    0    0 
_Cleaner_ 925517922  6  2 Bc  kf    -     -          390   20    0 
_Cleaner_ 925517939  6  1 Bc  kc    kf    -          370   20    0 
_Cleaner_ 925517984  6  6 f   -     -     -          350    0    0 
_Cleaner_ 925518035  6  5 f   -     -     -          350    0    0 
_Cleaner_ 925518073  6  4 cc  c     c     f          350   50    0 
_Cleaner_ 925518120  7  3 f   -     -     -          300    0    0 
_Cleaner_ 925518164  7  2 Bk  k     k     k          300   10   10 3s 8h
_Cleaner_ 925518188  8  1 Br  br    b     b          300  110  350 Tc 8c
_Cleaner_ 925518232  9  9 c   f     -     -          540   30    0 
_Cleaner_ 925518300  9  8 f   -     -     -          510    0    0 
_Cleaner_ 925518385  9  7 cc  k     c     f          510   60    0 
_Cleaner_ 925518479 10  6 f   -     -     -          450    0    0 
_Cleaner_ 925518537 10  5 cc  b     b     b          450   70  270 4c Tc
_Cleaner_ 925518620 10  4 cc  kcc   kf    -          650   70    0 
_Cleaner_ 925518731 10  4 cr  kfQ   -     -          580   40    0 
