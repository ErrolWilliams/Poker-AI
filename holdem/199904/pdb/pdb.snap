snap      923052775  2  2 Brc rr    rr    rr       20690  240  460 
snap      923052806  2  1 Brr brc   brc   br       20910  220  420 
snap      923052838  2  2 Brc rr    rr    r        21110  200  380 
snap      923052865  3  1 Br  b     -     -        21290   30   50 
snap      923052883  3  3 r   c     k     k        21310   30    0 5h 5s
snap      923052914  3  2 Bc  r     r     rc       21280  140    0 4d Jd
snap      923052959  3  1 BQ  -     -     -        21140    5    0 
snap      923055209  2  2 B   -     -     -        31580   10   15 
snap      923055210  2  1 Br  -     -     -        31585   20   30 
snap      923055212  2  2 B   -     -     -        31595   10   15 
snap      923055213  2  1 Br  -     -     -        31600   20   30 
snap      923055215  2  2 B   -     -     -        31610   10   15 
snap      923055216  2  1 Br  -     -     -        31615   20   30 
snap      923055217  2  2 B   -     -     -        31625   10   15 
snap      923055218  2  1 Br  -     -     -        31630   20   30 
snap      923055219  2  2 B   -     -     -        31640   10   15 
snap      923055220  2  1 Br  -     -     -        31645   20   30 
snap      923055221  2  2 B   -     -     -        31655   10   15 
snap      923055222  2  1 Br  -     -     -        31660   20   30 
snap      923055223  2  2 B   -     -     -        31670   10   15 
snap      923055224  2  1 Br  -     -     -        31675   20   30 
snap      923055225  2  2 B   -     -     -        31685   10   15 
snap      923055227  2  1 Br  -     -     -        31690   20   30 
snap      923055228  2  2 B   -     -     -        31700   10   15 
snap      923055229  2  1 Br  -     -     -        31705   20   30 
snap      923055230  2  2 B   -     -     -        31715   10   15 
snap      923055231  2  1 B   -     -     -        31720    5    0 3c Qh
snap      923055232  2  2 B   -     -     -        31715   10   15 
snap      923055235  2  1 B   -     -     -        31720    5    0 As 5c
snap      923055236  2  2 B   -     -     -        31715   10   15 
snap      923055239  2  1 B   -     -     -        31720    5    0 7c 3h
snap      923055240  2  2 B   -     -     -        31715   10   15 
snap      923055244  2  1 B   -     -     -        31720    5   10 4s Kd
snap      923055599  2  2 B   -     -     -        31740   10   15 
snap      923055600  2  1 Br  -     -     -        31745   20   30 
snap      923055602  2  2 B   -     -     -        31755   10   15 
snap      923055603  2  1 Br  -     -     -        31760   20   30 
snap      923055706  2  2 B   -     -     -        31770   10   15 
snap      923055707  2  1 Br  -     -     -        31775   20   30 
snap      923055708  2  2 B   -     -     -        31785   10   15 
snap      923055709  2  1 Br  -     -     -        31790   20   30 
snap      923055711  2  2 B   -     -     -        31800   10   15 
snap      923055712  2  1 Br  -     -     -        31805   20   30 
snap      923055713  2  2 B   -     -     -        31815   10   15 
snap      923055714  2  1 Br  -     -     -        31820   20   30 
snap      923055724  2  2 B   -     -     -        31830   10   15 
snap      923055725  2  1 Br  -     -     -        31835   20   30 
snap      923055727  2  2 B   -     -     -        31845   10   15 
snap      923055728  2  1 Br  -     -     -        31850   20   30 
snap      923055729  2  2 B   -     -     -        31860   10   15 
snap      923055730  2  1 Br  -     -     -        31865   20   30 
snap      923055731  2  2 B   -     -     -        31875   10   15 
snap      923055732  2  1 Br  -     -     -        31880   20   30 
snap      923055746  2  2 B   -     -     -        31890   10   15 
snap      923055747  2  1 Br  -     -     -        31895   20   30 
snap      923055749  2  2 B   -     -     -        31905   10   15 
snap      923055750  2  1 Br  -     -     -        31910   20   30 
snap      923055751  2  2 B   -     -     -        31920   10   15 
snap      923055752  2  1 Br  -     -     -        31925   20   30 
snap      923055753  2  2 B   -     -     -        31935   10   15 
snap      923055754  2  1 Br  -     -     -        31940   20   30 
snap      923055755  2  2 B   -     -     -        31950   10   15 
snap      923055756  2  1 Br  -     -     -        31955   20   30 
snap      923055758  2  2 B   -     -     -        31965   10   15 
snap      923055759  2  1 Br  -     -     -        31970   20   30 
snap      923055760  2  2 B   -     -     -        31980   10   15 
snap      923055761  2  1 Br  -     -     -        31985   20   30 
snap      923055762  2  2 B   -     -     -        31995   10   15 
snap      923055763  2  1 Br  -     -     -        32000   20   30 
snap      923055764  2  2 B   -     -     -        32010   10   15 
snap      923055765  2  1 Br  -     -     -        32015   20   30 
snap      923055767  2  2 B   -     -     -        32025   10   15 
snap      923055768  2  1 Br  -     -     -        32030   20   30 
snap      923055772  2  2 B   -     -     -        32040   10   15 
snap      923055773  2  1 Br  -     -     -        32045   20   30 
snap      923055778  2  2 B   -     -     -        32055   10   15 
snap      923055779  2  1 Br  -     -     -        32060   20   30 
snap      923055780  2  2 B   -     -     -        32070   10   15 
snap      923055781  2  1 Br  -     -     -        32075   20   30 
snap      923055783  2  2 B   -     -     -        32085   10   15 
snap      923055784  2  1 Br  -     -     -        32090   20   30 
snap      923055787  2  2 B   -     -     -        32100   10   15 
snap      923055788  2  1 Br  -     -     -        32105   20   30 
snap      923055789  2  2 B   -     -     -        32115   10   15 
snap      923055790  2  1 Br  -     -     -        32120   20   30 
snap      923055791  2  2 B   -     -     -        32130   10   15 
snap      923055792  2  1 Br  -     -     -        32135   20   30 
snap      923055793  2  2 B   -     -     -        32145   10   15 
snap      923055794  2  1 Br  -     -     -        32150   20   30 
snap      923055796  2  2 B   -     -     -        32160   10   15 
snap      923055797  2  1 Br  -     -     -        32165   20   30 
snap      923055798  2  2 B   -     -     -        32175   10   15 
snap      923055799  2  1 Br  -     -     -        32180   20   30 
snap      923055800  2  2 B   -     -     -        32190   10   15 
snap      923055801  2  1 Br  -     -     -        32195   20   30 
snap      923055802  2  2 B   -     -     -        32205   10   15 
snap      923055803  2  1 Br  -     -     -        32210   20   30 
snap      923055804  2  2 B   -     -     -        32220   10   15 
snap      923055805  2  1 Br  -     -     -        32225   20   30 
snap      923055806  2  2 B   -     -     -        32235   10   15 
snap      923055807  2  1 Br  -     -     -        32240   20   30 
snap      923055808  2  2 B   -     -     -        32250   10   15 
snap      923055809  2  1 Br  -     -     -        32255   20   30 
snap      923055810  2  2 B   -     -     -        32265   10   15 
snap      923055811  2  1 Br  -     -     -        32270   20   30 
snap      923055812  2  2 B   -     -     -        32280   10   15 
snap      923055813  2  1 Br  -     -     -        32285   20   30 
snap      923055814  2  2 B   -     -     -        32295   10   15 
snap      923055815  2  1 Br  -     -     -        32300   20   30 
snap      923055817  2  2 B   -     -     -        32310   10   15 
snap      923055818  2  1 Br  -     -     -        32315   20   30 
snap      923055820  2  2 B   -     -     -        32325   10   15 
snap      923055821  2  1 Br  -     -     -        32330   20   30 
snap      923055822  2  2 B   -     -     -        32340   10   15 
snap      923055823  2  1 Br  -     -     -        32345   20   30 
snap      923055824  2  2 B   -     -     -        32355   10   15 
snap      923055825  2  1 Br  -     -     -        32360   20   30 
snap      923055826  2  2 B   -     -     -        32370   10   15 
snap      923055827  2  1 Br  -     -     -        32375   20   30 
snap      923055828  2  2 B   -     -     -        32385   10   15 
snap      923055829  2  1 Br  -     -     -        32390   20   30 
snap      923055830  2  2 B   -     -     -        32400   10   15 
snap      923055831  2  1 Br  -     -     -        32405   20   30 
snap      923055833  2  2 B   -     -     -        32415   10   15 
snap      923055834  2  1 Br  -     -     -        32420   20   30 
snap      923055835  2  2 B   -     -     -        32430   10   15 
snap      923055836  2  1 Br  -     -     -        32435   20   30 
snap      923055837  2  2 B   -     -     -        32445   10   15 
snap      923055838  2  1 Br  -     -     -        32450   20   30 
snap      923055839  2  2 B   -     -     -        32460   10   15 
snap      923055840  2  1 Br  -     -     -        32465   20   30 
snap      923055841  2  2 B   -     -     -        32475   10   15 
snap      923055842  2  1 Br  -     -     -        32480   20   30 
snap      923055843  2  2 B   -     -     -        32490   10   15 
snap      923055844  2  1 Br  -     -     -        32495   20   30 
snap      923055845  2  2 B   -     -     -        32505   10   15 
snap      923055846  2  1 Br  -     -     -        32510   20   30 
snap      923055847  2  2 B   -     -     -        32520   10   15 
snap      923055848  2  1 Br  -     -     -        32525   20   30 
snap      923055849  2  2 B   -     -     -        32535   10   15 
snap      923055850  2  1 Br  -     -     -        32540   20   30 
snap      923055851  2  2 B   -     -     -        32550   10   15 
snap      923055852  2  1 Br  -     -     -        32555   20   30 
snap      923055853  2  2 B   -     -     -        32565   10   15 
snap      923055854  2  1 Br  -     -     -        32570   20   30 
snap      923055855  2  2 B   -     -     -        32580   10   15 
snap      923055856  2  1 Br  -     -     -        32585   20   30 
snap      923055857  2  2 B   -     -     -        32595   10   15 
snap      923055858  2  1 Br  -     -     -        32600   20   30 
snap      923055859  2  2 B   -     -     -        32610   10   15 
snap      923055860  2  1 Br  -     -     -        32615   20   30 
snap      923055861  2  2 B   -     -     -        32625   10   15 
snap      923055862  2  1 Br  -     -     -        32630   20   30 
snap      923055863  2  2 B   -     -     -        32640   10   15 
snap      923055864  2  1 Br  -     -     -        32645   20   30 
snap      923055865  2  2 B   -     -     -        32655   10   15 
snap      923055866  2  1 Br  -     -     -        32660   20   30 
snap      923055867  2  2 B   -     -     -        32670   10   15 
snap      923055868  2  1 Br  -     -     -        32675   20   30 
snap      923055869  2  2 B   -     -     -        32685   10   15 
snap      923055870  2  1 Br  -     -     -        32690   20   30 
snap      923055871  2  2 B   -     -     -        32700   10   15 
snap      923055872  2  1 Br  -     -     -        32705   20   30 
snap      923055873  2  2 B   -     -     -        32715   10   15 
snap      923055874  2  1 B   -     -     -        32720    5   10 8c Jh
snap      923055886  2  2 B   -     -     -        32725   10   15 
snap      923055887  2  1 Br  -     -     -        32730   20   30 
snap      923055888  2  2 B   -     -     -        32740   10   15 
snap      923055889  2  1 Br  -     -     -        32745   20   30 
snap      923055890  2  2 B   -     -     -        32755   10   15 
snap      923055891  2  1 Br  -     -     -        32760   20   30 
snap      923055892  2  2 B   -     -     -        32770   10   15 
snap      923055893  2  1 Br  -     -     -        32775   20   30 
snap      923055894  2  2 B   -     -     -        32785   10   15 
snap      923055895  2  1 Br  -     -     -        32790   20   30 
snap      923055897  2  2 B   -     -     -        32800   10   15 
snap      923055898  2  1 Br  -     -     -        32805   20   30 
snap      923055899  2  2 B   -     -     -        32815   10   15 
snap      923055900  2  1 Br  -     -     -        32820   20   30 
snap      923055901  2  2 B   -     -     -        32830   10   15 
snap      923055902  2  1 Br  -     -     -        32835   20   30 
snap      923055903  2  2 B   -     -     -        32845   10   15 
snap      923055904  2  1 Br  -     -     -        32850   20   30 
snap      923055905  2  2 B   -     -     -        32860   10   15 
snap      923055906  2  1 Br  -     -     -        32865   20   30 
snap      923055907  2  2 B   -     -     -        32875   10   15 
snap      923055908  2  1 Br  -     -     -        32880   20   30 
snap      923055909  2  2 B   -     -     -        32890   10   15 
snap      923055910  2  1 BQ  -     -     -        32895    5    0 
